libandroid-navigation-ui/src/main/java/com/mapbox/services/android/navigation/ui/v5/NavigationView.java

```diff
@@ -406,7 +406,7 @@ private void init() {
     InstructionLoader.getInstance().initialize(getContext());
     inflate(getContext(), R.layout.navigation_view_layout, this);
     bind();
-    initViewModels();
+    initNavigationViewModel();
```

Comment:
For consistency, could we rename `init` methods to `initialize`? Saw that we're using `initialize` nomenclature in `navigationViewModel` for example.